commands/__init__.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+
+def discover_commands(logging):
+    """ This function will discover files that exist in the CWD.
+        It will import those files and generate a dict with all
+        of the predefined command aliases for each command. This
+        will be templated and should be easy to understand.
+    """
+    import os
+    import sys
+    import importlib
+    _mypath = os.path.abspath(__file__)
+    MYWORKDIR = os.path.dirname(_mypath)
+    logging.debug(MYWORKDIR)
+    _removal_list = ['__init__.py', '__pycache__', 'template.py']
+    file_list = os.listdir(MYWORKDIR)
+    logging.info("Remove files in the ban list - files to not import.")
+    for item in _removal_list:
+        logging.debug("removing {0}".format(item))
+        file_list.remove(item)
+    commands = {}
+    logging.debug("Checking files in file list for py extention.")
+    for item in file_list:
+        _aliases = {}
+        logging.debug("Checking file: {0}".format(item))
+        if item.split('.')[1] == 'py':
```

Comment:
This assumes no other dots in the filename

Reply:
Correct, python3 won't import files names that have dots in them. Python treats dot delimitation as a module (subdirectory). 

Example.
`dwalton@Derricks-MacBook-Pro:~/tmp$ ls 
start.test.py
dwalton@Derricks-MacBook-Pro:~/tmp$ python3
Python 3.7.4 (default, Sep  7 2019, 18:27:02) 
[Clang 10.0.1 (clang-1001.0.46.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import start.test
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'start' `